test/mailers/notification_mailer_test.rb

```diff
@@ -27,4 +30,16 @@ def test_mail_gets_delivered
     assert_equal(['administrator@auction.test'], email.to)
     assert_equal('Daily summary for 2010-07-04', email.subject)
   end
+
+  def test_sends_auctioned_domain_list_to_subscribers
+    perform_enqueued_jobs do
+      DailyBroadcastAuctionsJob.perform_now
+    end
+
+    assert_not(ActionMailer::Base.deliveries.empty?)
+    email = ActionMailer::Base.deliveries.last
+    assert_equal(['user@auction.test'], email.to)
```

Comment:
It's not clear where does this value come from. Can we explain it (with code) in the "given" phase?